scripts/tagging/gitinterface.py

```diff
@@ -12,7 +13,7 @@
 from operator import itemgetter
 
 from helperfunctions import parseForReleaseNotes, curl2Json, \
-                            authorMapping, versionComp
+                            authorMapping, versionComp, versionToKey
 
 from parseversion import Version
```

Comment:
Alternatively via a relative import 
```suggestion
from .parseversion import Version
```

Otherwise this fails for me with python3(.8.5). This is probably needed also in a few other places in `tagging` that are not in this PR.